lib/CMakeLists.txt

```diff
@@ -251,8 +251,7 @@ endif()
 if (WIN32)
     set(LIBFACTER_INSTALL_DESTINATION bin)
 else()
-    # TODO: lib64 for certain operating systems?
-    set(LIBFACTER_INSTALL_DESTINATION lib)
+    set(LIBFACTER_INSTALL_DESTINATION ${CMAKE_INSTALL_LIBDIR})
```

Comment:
facter.rb.in may also need to be updated for this. At least, Travis builds are failing related to finding libfacter.so